README.md

```diff
@@ -2,44 +2,65 @@
 # Emberfall
 
 <img align="right" width="200" src="./emberfall-logo.png">
+
 **HTTP smoke testing made easy.**
 
 Simply declare a list of URLs and their expected response values, and Emberfall will test, compare, and report which URLs fail along with details of what expectations were not met.
 
 ## Configuring Tests
+Tests are merely a list of request objects, each with a url, method, headers to be sent, and an expects field. With the `expects` field you can define the status code, body contents (as a string), and any headers (as strings) that should be present in the response. If anything expected is not present or not equal to the defined value `emberfall` will exit with a non-zero code.
 
-Tests are defined in a simple YAML document like the one below:
+The YAML tests config can be provided in two ways:
+- as a file: `emberfall --config path/to/config.yaml`
+- piped to stdin: `echo $EMBERFALL_CONFIG | emberfall --config -` 
+
+Tests are defined in a simple YAML document that defines the following keys:
 ```yaml
-commonHeaders: &commonHeaders
-  authorization: "<authorization header>"
 tests:
-- url:
-  method:
+- url: string
+  method: string #a supported HTTP method such as GET, POST, PUT, DELETE, etc...
   expect:
+    status: int #a supported HTTP status code such as 200,201,301,400,404, etc...
+    body: string # optional
+    headers: #optional
+      # key:value where header key is expected to be present in the response
+```
+
+### Basic Tests Configuration
+The following example is the minimum configuration required to execute Emberfall tests. This performs a single test against an API endpoint, expecting to receive a 401 unauthorized response:
+```yaml
+tests:
 - url: http://localhost:3000/api/v1/users/current
   method: GET
   expect:
     status: 401
     body: "unauthorized"
-
+```
```

Comment:
Is there something going on with the rendering of the yaml here?

Reply:
It may be that a newline is needed to render it correctly on GH, as on L27-29